alf/experience_replayers/replay_buffer.py

```diff
@@ -270,10 +298,27 @@ def get_batch(self, batch_size, batch_length):
                         non-zero priorities in the buffer.
         """
         with alf.device(self._device):
+            recent_batch_size = 0
+            if self._recent_data_ratio > 0:
+                avg_size = self.total_size / float(
+                    self._num_envs) - batch_length + 1
+                if (avg_size * self._recent_data_ratio >
+                        self._recent_data_steps):
```

Comment:
is this if condition for the case where there is enough recent data?
Shouldn't this be
```
if (batch_size / num_envs - batch_length + 1) * recent_data_ratio < recent_data_steps and (
    avg_size > recent_data_steps)
```